gruntz.tex

```diff
@@ -1,55 +1,98 @@
-We first define comparability classes by calculating $L$:
+SymPy calculates limits using the Gruntz algorithm, as described in
+\cite{Gruntz1996limits}. The basic idea is as follows: any limit can be
+converted to a limit $\lim_{x\to\infty} f(x)$ by substitutions like
+$x\to{1\over x}$.
+Then we identify in $f(x)$ the most varying subexpression $\omega$ (that
```

Comment:
Ok, I converted the whole PR to passive form in a3226e6f8fcb8c6ef3a8d7afefa3f51018659542.